cursors.yaml

```diff
@@ -273,7 +273,7 @@ Cursors:
 			Start: 73
 			Length: 5
 		powerdown-blocked:
-			Start: 52
+			Start: 345
```

Comment:
I think it may be worse repeating `339` here without animation, or `342` to show a greyed out version. This looks like an unrelated movement block cursor which is a bit irritating. Feels like I am not in powerdown mode.